tests/openvino/pot/quantization/test_sanity.py

```diff
@@ -25,7 +25,7 @@
 
 OMZ_MODELS = [
     ('resnet-18-pytorch', 'imagenette2-320', {'accuracy@top1': 0.777, 'accuracy@top5': 0.949}),
-    ('mobilenet-v3-small-1.0-224-tf', 'imagenette2-320', {'accuracy@top1': 0.737, 'accuracy@top5': 0.923}),
+    ('mobilenet-v3-small-1.0-224-tf', 'imagenette2-320', {'accuracy@top1': 0.744, 'accuracy@top5': 0.922}),
```

Comment:
`inplace_statistics = True` is changed metrics of the mobilenet-v3-small-1.0-224-tf. As far I know `True` is default value for POT `inplace_statistics`, @KodiaqQ could you comment why feeding the default value is influence on result metrics of the quantized model?